smbios.adoc

```diff
@@ -29,67 +29,79 @@ on top of cite:[SMBIOS], and is optional and recommended for BRS-B. Additional n
 [25:0] in RISC-V Machine ISA Register (**misa** CSR). Bits set to one mean the
 certain extensions of instruction set are supported on this hart.
-| 27h| 000Ah| Privilege Level Supported| BYTE| Varies| The privilege levels
+| 27h| 0100h| Privilege Level Supported| BYTE| Varies| The privilege levels
 supported by this RISC-V Hart. +
 Bit 0 Machine Mode +
 BIT 1 Reserved +
 BIT 2 Supervisor Mode +
 Bit 3 User Mode +
 BIT 6:4 Reserved +
 BIT 7 Debug Mode
-| 28h| 000Ah| Machine Exception Trap Delegation Information| QWORD| Varies|
+| 28h| 0100h| Machine Exception Trap Delegation Information| QWORD| Varies|
 Bit set to one means the corresponding exception is delegated to supervisor execution
 environment. Otherwise, supervisor execution environment must register the event
 handler in Machine-Mode for the certain exceptions through environment call.
-| 30h| 000Ah| Machine Interrupt Trap Delegation Information| QWORD| Varies|
+| 30h| 0100h| Machine Interrupt Trap Delegation Information| QWORD| Varies|
 Bit set to one means the corresponding interrupt is delegated to supervisor execution
 environment. Otherwise, supervisor execution environment must register the event
 handler in Machine-Mode for the certain interrupts through environment.
-| 38h| 000Ah| The register width (XLEN)| BYTE| ENUM| The width of register
+| 38h| 0100h| The register width (XLEN)| BYTE| ENUM| The width of register
 supported by this RISC-V Hart
-| 39h| 000Ah| Machine Mode native base integer ISA width (M-XLEN)| BYTE| ENUM|
+| 39h| 0100h| Machine Mode native base integer ISA width (M-XLEN)| BYTE| ENUM|
 The width (See below) of Machine Mode native base integer ISA supported by this RISC-V
 Hart
-| 3Ah| 000Ah| Reserved| BYTE| ENUM| Placeholder for Hypervisor Mode
-| 3Bh| 000Ah| Supervisor Mode native base integer ISA width (S-XLEN)| BYTE| ENUM|
+| 3Ah| 0100h| Reserved| BYTE| ENUM| Placeholder for Hypervisor Mode
+| 3Bh| 0100h| Supervisor Mode native base integer ISA width (S-XLEN)| BYTE| ENUM|
 The width (See below) of Supervisor Mode native base integer ISA supported by this RISC-V
 Hart
-| 3Ch| 0000Ah| User Mode native base integer ISA width (U-XLEN)| BYTE| ENUM| The
+| 3Ch| 00100h| User Mode native base integer ISA width (U-XLEN)| BYTE| ENUM| The
 width (See below) of the User Mode native base integer ISA supported by this RISC-V Hart
 |===
 
+[[smbios-psd-ver]]
+=== Processor-Specific Data Structure Versioning
+
+The processor-specific data structure begins with a revision field to allow for future extensibility in a backwards-compatible manner.
+
+The minor revision is to be incremented anytime new fields are added in a backwards-compatible manner. The major revision is to be incremented on backwards-incompatible changes.
+
+[cols="1,1,1,1,3", width=95%, align="center", options="header"]
+|===
+| Version | Bits 15:8+
+Major revision
+| Bits 7:0+
+Minor revision
+| Combined | Description
+| v1.0 | 01h | 00h | 0100h | First BRS-defined definition
```

Comment:
My other comment is now addressed.  Should we put this first at the beginning of the section like it was before?

Reply:
Personally I think its fine where it is. There's a link to the section from the table so you can easily jump to it.